src/coord/catalog.rs

```diff
@@ -654,6 +664,32 @@ impl Catalog {
         }
     }
 
+    /// Gets GlobalIds of temporary items to be created, checks for name collisions
+    /// within a connection id.
+    fn temporary_ids(&mut self, ops: &[Op]) -> Result<Vec<GlobalId>, Error> {
+        let mut creating = HashSet::new();
+        let mut temporary_ids = Vec::new();
+        for op in ops.iter() {
+            if let Op::CreateItem { id, name, item } = op {
+                if let CatalogItem::View(view) = item {
+                    if let Some(conn_id) = view.conn_id {
+                        if self.item_exists_in_temp_schemas(conn_id, name.item.clone())
+                            | creating.contains(&(conn_id, name.item.clone()))
```

Comment:
I think you want `||` here instead of `|`. (I don't think the lack of short-circuiting really matters here, but it's a bit surprising.)